Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Dockerfile to build the application on Render
+FROM maven:3.8.5-openjdk-17
+COPY /src /app/src
+COPY pom.xml /app/
+WORKDIR /app
+# Disable tests on Render
+RUN mvn clean package -DskipTests
+EXPOSE 8080
```

Comment:
I haven't really looked into render, but shouldn't you use a [multi-stage build](https://docs.docker.com/build/building/multi-stage/#use-multi-stage-builds)? Use the maven image for building, then use a e.g., `eclipse-temurin:17-jre-alpine` as the base for the run image copying over the jar file.